scripts/install/requestrr.sh

```diff
@@ -0,0 +1,178 @@
+    "Password": "",
+    "PrivateKey": "[PRIVATEKEY]"
+  },
+  "ChatClients": {
+    "Discord": {
+      "BotToken": "",
+      "ClientId": "",
+      "StatusMessage": "!help",
+      "TvShowRoles": [],
+      "MovieRoles": [],
+      "MonitoredChannels": [],
+      "EnableRequestsThroughDirectMessages": false,
+      "AutomaticallyNotifyRequesters": true,
+      "NotificationMode": "PrivateMessages",
+      "NotificationChannels": [],
+      "AutomaticallyPurgeCommandMessages": false,
+      "DisplayHelpCommandInDMs": true
+    }
+  },
+  "DownloadClients": {
+    "Ombi": {
+      "Hostname": "",
+      "Port": 3579,
+      "ApiKey": "",
+      "ApiUsername": "",
+      "BaseUrl": "",
+      "UseSSL": false,
+      "Version": "3"
+    },
+    "Overseerr": {
+      "Hostname": "",
+      "Port": 5055,
+      "ApiKey": "",
+      "DefaultApiUserID": "",
+      "UseSSL": false,
+      "Version": "1"
+    },
+    "Radarr": {
+      "Hostname": "",
+      "Port": 7878,
+      "ApiKey": "",
+      "BaseUrl": "",
+      "MovieProfileId": "1",
+      "MovieRootFolder": "",
+      "MovieMinimumAvailability": "",
+      "MovieTags": [],
+      "AnimeProfileId": "1",
+      "AnimeRootFolder": "",
+      "AnimeMinimumAvailability": "",
+      "AnimeTags": [],
+      "SearchNewRequests": true,
+      "MonitorNewRequests": true,
+      "UseSSL": false,
+      "Version": "2"
+    },
+    "Sonarr": {
+      "Hostname": "",
+      "Port": 8989,
+      "ApiKey": "",
+      "BaseUrl": "",
+      "TvProfileId": "1",
+      "TvRootFolder": "",
+      "TvTags": [],
+      "TvLanguageId": "1",
+      "TvUseSeasonFolders": true,
+      "AnimeProfileId": "1",
+      "AnimeRootFolder": "",
+      "AnimeTags": [],
+      "AnimeLanguageId": "1",
+      "AnimeUseSeasonFolders": true,
+      "SearchNewRequests": true,
+      "MonitorNewRequests": true,
+      "UseSSL": false,
+      "Version": "3"
+    }
+  },
+  "BotClient": {
+    "Client": "",
+    "CommandPrefix": "!"
+  },
+  "Movies": {
+    "Client": "Disabled",
+    "Command": "movie"
+  },
+  "TvShows": {
+    "Client": "Disabled",
+    "Command": "tv",
+    "Restrictions": "None"
+  },
+  "Port": 4545,
+  "BaseUrl" : "/requestrr",
+  "Version": "1.12.0"
+}
+CFG
+
+echo_progress_done "Requestrr config applied."
+
+echo_progress_start "Installing Systemd service"
+cat > /etc/systemd/system/requestrr.service << EOF
+[Unit]
+Description=Requestrr Daemon
+After=syslog.target network.target
+
+[Service]
+User=requestrr
+Type=simple
+WorkingDirectory=/opt/requestrr/
+ExecStart=/opt/requestrr/Requestrr.WebApi
+TimeoutStopSec=20
+KillMode=process
+Restart=on-failure
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+if [[ -f /install/.nginx.lock ]]; then
+    echo_progress_start "Installing nginx configuration"
+    #TODO what is this sleep here for? See if this can be fixed by doing a check for whatever it needs to
+    sleep 10
+    bash /usr/local/bin/swizzin/nginx/requestrr.sh
+    systemctl daemon-reload
```

Comment:
why's there a daemon reload here? if the nginx scriptis modifying the service then that should be there instead

Reply:
looks like it's not being modified, no need for it at all i think

Reply:
removed the daemon-reload and sleep 10